data-processing/scripts/run_pipeline.py

```diff
@@ -288,6 +289,14 @@ def run_commands_for_arxiv_ids(
             + "(see https://www.postgresql.org/docs/9.2/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS)."
         ),
     )
+    parser.add_argument(
+        "--database-create-tables",
+        type=bool,
+        default=False,
```

Comment:
This behavior can also be expressed with the shorthand `action="store_true"`

Reply:
Yep! I had a second commit yesterday to use that https://github.com/allenai/scholarphi/pull/155/commits/22be9915f6ef83eddce505d8101d5ee74d3b253b